tests/runtime/test_grpc_server.py

```diff
@@ -953,6 +954,28 @@ def test_runtime_status_ok_response(runtime_grpc_server):
     assert actual_response.numericRuntimeVersion == 0
 
 
+# TODO: remove input of TrainingInfoRequest
+def test_runtime_info_ok_response(sample_task_model_id, runtime_grpc_server):
```

Comment:
Currently failing with error `AttributeError: 'dict' object has no attribute 'GetServerInfo'` on [this line](https://github.com/caikit/caikit/compare/main...anhuong:caikit:server-info?expand=1#diff-b272a1480be61d2c6958dbb793953698ffa95dd8bfae011d5ba9ad3e3a2369ebR151-R152) which makes a lot of sense since the method I attached is not a Class that implements `GetServerInfo` . I see how it’s done for TrainingManagement but what I don’t understand is how HTTP server implements the TrainingManagement portions? Because I thought I had to implement it in `server_base` so that I could reference it within each server like [here](https://github.com/caikit/caikit/compare/main...anhuong:caikit:server-info?expand=1#diff-8c48c03bdb3588a96567eff810341f816e17768a82d89ab2a501e7d8649abc94R266)

Reply:
--> moved from `server_base` into own servicer implementation

Reply:
@anhuong I don't think the http server supports the `TrainingManagementService`